{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "a412feb8_ec011bb0",
        "filename": "zuul/driver/sql/sqlconnection.py",
        "patchSetId": 1
      },
      "lineNbr": 392,
      "author": {
        "id": 27582
      },
      "writtenOn": "2023-08-07T09:44:17Z",
      "side": 1,
      "message": "I think instead of continuing as if the version was correct, we should wait until the alembic version was updated.",
      "revId": "23778509e7612c6eb63398d434f943ea16eeedf3",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}